heltour/tournament/tests/testutils.py

```diff
@@ -1,5 +1,5 @@
-from heltour.tournament.models import (League, LonePlayerScore, Player, Registration, Season, 
```

Comment:
`Round` doesn't appear to be used in this file.

Reply:
it's used in `get_rounds` - which itself is not used anywhere, yet, but i am already using it in a local test that is not in this pr yet.